sql/core/src/main/scala/org/apache/spark/sql/DataFrameNaFunctions.scala

```diff
@@ -130,6 +130,13 @@ final class DataFrameNaFunctions private[sql](df: DataFrame) {
   /**
    * Returns a new [[DataFrame]] that replaces null or NaN values in numeric columns with `value`.
    *
+   * @since 2.1.0
+   */
+  def fill(value: Long): DataFrame = fill(value, df.columns)
+
+  /**
+   * Returns a new [[DataFrame]] that replaces null or NaN values in numeric columns with `value`.
```

Comment:
Could I ask change `[[DataFrame]]` to `` `DataFrame` ``?  It seems the `DataFrame` is unrecognisable via unidoc/genjavadoc (see https://github.com/apache/spark/pull/16013) which ends up with documentation build failure with Java 8.